tests/integration/modules/test_cp.py

```diff
@@ -64,6 +64,7 @@ def test_get_file_templated_paths(self):
             self.assertIn('Gromit', data)
             self.assertNotIn('bacon', data)
 
+    @skipIf(__salt_system_encoding__ == 'US-ASCII', 'Requires a unicode locale, i.e. en_US.UTF-8')
```

Comment:
What would be the best way to do this?

I pass this to salt.utils.to_bytes and it explodes because it can't encode it as 'latin-1'